dolphin/drivers/dellemc_storage/vmax/vmax.py

```diff
@@ -64,33 +52,33 @@ def get_storage_registry():
 
     def get_storage(self, context):
 
-        self._check_connection(context)
         # Get the VMAX model
-        model = client.get_model(self.conn, self.symmetrix_id)
+        model = self.client.get_model(self.symmetrix_id)
 
         # Get Storage details for capacity info
-        storg_info = client.get_storage_capacity(self.conn, self.symmetrix_id)
-        total_cap = storg_info['usable_total_tb']
-        used_cap = storg_info['usable_used_tb']
+        storg_info = self.client.get_storage_capacity(self.symmetrix_id)
+        total_cap = storg_info.get('usable_total_tb')
+        used_cap = storg_info.get('usable_used_tb')
+        free_cap = total_cap - used_cap
 
         storage = {
             'id': self.storage_id,
             'name': '',
             'vendor': 'Dell EMC',
```

Comment:
I think we should return vendor string which is exactly used to get driver.
        'dolphin.storage.drivers': [
            'fake_storage fake_driver = dolphin.drivers.fake_storage:FakeStorageDriver',
            'dellemc vmax = dolphin.drivers.dellemc_storage.vmax.vmax:VMAXStorageDriver'

Reply:
We return 'model' which is returned from Array (Eg: VMAX250F). So I think, 'vendor' too, can be different.